content/en/docs/deployment/installation/developer-portal.md

```diff
@@ -100,3 +105,27 @@ Customize the `open-banking-developer-portal/values.yaml` file as follows.
 5. Check the different user interfaces:
    * _Developer Portal home page_: `https://developer-portal.<domain-name>`. If the APIM Helm charts are successfully deployed, you should see APIs on the API Catalog (click the API tab).
    * _Joomla admin interface_: `https://developer-portal.<domain-name>/administrator`. Login with username _apiadmin_ and password _apiAdminPwd!_.
+
+## Post Install steps
+
+1. Customizing the brand.
+   If needed, you can easily brand the developer portal with your logo and brand colors.
+   Connect the the adminstrator UI and navigate to:
+   ```
+       Extentions > Templates > Style -  Choose a Style profile, then click `Theme` tab, next click `Theme Magic` button top of screen.
+   ```
+   Key Colors:
+   ```
+       Brand Color: The base color of the site
+   ```
+   Basic Colors:
+   ```
+       @page-text-color: The base text color of the site
+   ```
+   Save the theme by clicking the arrow button next to preview button to `Save As`
+2. Apply Theme
+   To apply them exit out of the Theme Magic (Back to Administrator) then change the `Theme` dropdown to select your new theme.
+3. Customizing the Logo
+   To change the logo use the `Logo Image` Selector under the same Theme tab.
+4. Saving the changes
+   Theme Magice only needs to be done once but to apply the theme saving will need to be done for each style you want to apply this to.
```

Comment:
```suggestion
Theme Magic only needs to be done once but to apply the theme saving will need to be done for each style you want to apply this to.
```
This needs to be rewritten, but I'm not sure what is means. Maybe you can send me more details and screen shots in Teams and I can figure out how to write it to be more clear.